build/scripts/post-release.psm1

```diff
@@ -0,0 +1,95 @@
+$gitHubBotUserName="github-actions[bot]"
+$gitHubBotEmail="41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Just for curiosity - what is the source of the email? Is it documented somehow?

Reply:
That is the GitHub "noreply" email address format which is actually supported for all users. Wish I would have known that before 🤣

As far as the ID: https://[api.github.com/users/github-actions%5Bbot%5D](https://api.github.com/users/github-actions%5Bbot%5D)

A discussion: https://github.com/orgs/community/discussions/26560

Reply:
Thanks!

Reply:
Here is a test run: https://github.com/CodeBlanch/opentelemetry-dotnet-contrib/actions/runs/9099109966/job/25010904161

Here is the release it created: https://github.com/CodeBlanch/opentelemetry-dotnet-contrib/releases/tag/Exporter.OneCollector-1.9.0

Reply:
Here is a test run for AspNet stuff: https://github.com/CodeBlanch/opentelemetry-dotnet-contrib/actions/runs/9099945737/job/25013678335

Here is the release it created: https://github.com/CodeBlanch/opentelemetry-dotnet-contrib/releases/tag/Instrumentation.AspNet-1.9.0-alpha.1